app/components/graph-editor/canvas.ts

```diff
@@ -77,14 +77,40 @@ export function drawQuadraticLine(
 }
 
 /**
+ * drawCubicLine  
+ *   Draws a cubic bezier line between two points.
+ */
+export function drawCubicLine(
```

Comment:
This comment seems kind of petty. Please refer to the third bullet point of the commit message for b9f0ee4.